app/src/main/java/com/google/samples/apps/sunflower/GardenFragment.kt

```diff
@@ -42,10 +55,26 @@ class GardenFragment : Fragment() {
         return binding.root
     }
 
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+
+        requireActivity().addOnBackPressedCallback {
```

Comment:
For clean code, I see `add` op, maybe I am so restricted, is it ok to do `remvoe` at *destroy* place?

I mean , let `selectionTracker` as `OnBackPressedCallback` is more better, when I do.

Reply:
I think I understand. Let me know what you think with when I push the next revision. :)